src/components/controls/buttons/Button.js

```diff
@@ -214,6 +223,8 @@ Button.propTypes = {
   block: PropTypes.bool,
   /** Render with the alternative styling */
   alternative: PropTypes.bool,
+  /** Decides whether the button colors should enforce accessbility AA standards */
+  enforceAccessibility: PropTypes.bool,
```

Comment:
I know the documentation helps make it clear what this prop is for, but maybe we should use a different name than `enforceAccessibility`. The naming is a bit awkward, sort of implying that accessibility of the button can be flipped on and off, not necessarily pertaining to color contrast.